VS2017/Source/tests/TestTerrainGeneration.cpp

```diff
@@ -124,14 +124,6 @@ namespace test {
 
 	void TestTerrainGeneration::OnImGuiRender()
 	{
-		ImGui::SliderFloat3("Base Translation", &m_BaseTranslation.x, -50.0f, 50.0f);
-		ImGui::SliderFloat("Base Angle", &m_BaseAngle, 0.0f, 360.0f);
-		if (ImGui::Button("Reset Angle")) {
-			m_BaseAngle = 0.0f;
-			m_BaseRotation = glm::vec3(0.0f, 0.0f, 0.0f);
-		}
-		ImGui::SliderFloat3("Base Rotation", &m_BaseRotation.x, -1.0f, 1.0f);
-		ImGui::SliderFloat3("Base Scale", &m_BaseScale.x, 0.0f, 2.0f);
 		ImGui::SliderFloat3("Camera", &m_CameraPosition.x, -100.0f, 100.0f);
```

Comment:
could we also remove the camera option since that is handled by the mouse and keyboard movement?

Reply:
agreed, removed